xlators/features/locks/src/reservelk.c

```diff
@@ -96,6 +96,28 @@ __reservelk_conflict(xlator_t *this, pl_inode_t *pl_inode, posix_lock_t *lock)
     return ret;
 }
 
+int
+_pl_verify_reservelk(xlator_t *this, pl_inode_t *pl_inode, posix_lock_t *lock,
+                     const int can_block)
+{
+    int ret = 0;
+
+    if (__reservelk_conflict(this, pl_inode, lock)) {
+        lock->blocked = can_block;
+        list_add_tail(&lock->list, &pl_inode->blocked_calls);
+        ret = -1;
+        gf_log(this->name, GF_LOG_TRACE,
```

Comment:
This log is now done inside the lock. A logged message can take a non-negligible amount of time. We should try to avoid it.